src/main/resources/driven-adapter/dynamo-db/dynamo-db-template-adapter.java.mustache

```diff
@@ -1,19 +1,39 @@
 package {{package}}.dynamodb;
 
-import {{package}}.dynamodb.helper.TemplateAdapterOperations;
+import co.com.bancolombia.dynamodb.helper.TemplateAdapterOperations;
```

Comment:
Should keep the dynamic root package [like](https://github.com/bancolombia/scaffold-clean-architecture/pull/218/files#:~:text=package%20%7B%7Bpackage-,%7D,-%7D.dynamodb.helper%3B)

Reply:
thanks

Reply:
fixed